queries/boundaries.jinja2

```diff
@@ -112,27 +112,4 @@ FROM buffered_land
 WHERE
   {{ bounds['polygon']|bbox_filter('the_geom',3857) }}
 
-UNION ALL
```

Comment:
Yes, that is the intention. The boundary line data was breaking the admin boundaries post-process code where it attempts to assign left/right properties. Lines have no well-defined topological left/right, and that was breaking the code.

The cases where we had tests on linear data turned out to be replaceable with polygonal data. If that isn't the case generally, then we need to find more tests.